.github/workflows/build-pyodide-package.yml

```diff
@@ -0,0 +1,108 @@
+        wget -nc "${PYODIDE_RELEASE_URL}/${PYODIDE_VERSION}/${XBUILDENV_BASENAME}"
+        tar -xvf ${XBUILDENV_BASENAME}
+        rm ${XBUILDENV_BASENAME}
+    - name: read-makefile
+      run: |
+        make -f xbuildenv/pyodide-root/Makefile.envs .output_vars | grep 'PY' | tee -a $GITHUB_ENV
+    - name: get-emsdk
+      run: |
+        wget https://github.com/emscripten-core/emsdk/archive/refs/heads/main.zip
+        unzip main.zip
+        rm main.zip
+    - name: setup-emsdk
+      run: |
+        ./emsdk-main/emsdk install ${{env.PYODIDE_EMSCRIPTEN_VERSION}}
+        ./emsdk-main/emsdk activate ${{env.PYODIDE_EMSCRIPTEN_VERSION}}
+    - name: build-modcc-host
+      run: |
+        mkdir -p arbor_src/build_modcc_host
+        cd arbor_src/build_modcc_host
+        cmake .. -DARB_USE_BUNDLED_LIBS=ON
+        make modcc -j 8
+    - name: build-libarbor-emscripten
+      run: |
+        ABS_TOOLCHAIN=$(realpath emsdk-main/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake)
+        ABS_MODCC=$(realpath arbor_src/build_modcc_host/bin/modcc)
+        sed -i.bak -E \
+            '/only been tested|nonexistent_file|unbalanced brace/d' \
+            arbor_src/ext/random123/include/Random123/features/gccfeatures.h
+        mkdir -p arbor_src/build_arborlib_em
+        cd arbor_src/build_arborlib_em
+        cmake .. \
+            --toolchain "${ABS_TOOLCHAIN}" \
+            -DCMAKE_INSTALL_PREFIX=$(realpath prefix) \
+            -DARB_USE_BUNDLED_LIBS=ON \
+            -DARB_MODCC="${ABS_MODCC}" \
+            -DARB_ARCH=generic
+        make -j 8
+        make install
+    - name: build-arbor-python-so-emscripten
+      run: |
+        ABS_XBUILDENV_INCLUDE=$(realpath xbuildenv/pyodide-root${{ env.CPYTHONINSTALL }}/include/python${{ env.PYMAJOR }}.${{ env.PYMINOR }})
+        ABS_EM_ARBOR=$(realpath arbor_src/build_arborlib_em/prefix)
+        PY_VERSION_FLAT=${{ env.PYMAJOR }}${{ env.PYMINOR }}
+        ARBOR_PYLIB_OUTPUT=arbor/_arbor.cpython-${PY_VERSION_FLAT}-wasm32-emscripten.so
+        ABS_ARBOR_PYLIB_OUTPUT=$(pwd)/$ARBOR_PYLIB_OUTPUT
+        source emsdk-main/emsdk_env.sh
+        mkdir -p arbor arbor.dist-info
+        mkdir -p arbor_src/build_python_lib_em
+        cd arbor_src/build_python_lib_em
+        time em++ \
+            ../python/*.cpp \
+            -I ../ext/pybind11/include/ \
+            -I "${ABS_XBUILDENV_INCLUDE}" \
+            -I "${ABS_EM_ARBOR}/include" \
+            -std=c++20 \
+            -I "${ABS_EM_ARBOR}/lib/libarbor*" \
+            -o "${ABS_ARBOR_PYLIB_OUTPUT}" \
+            -shared -fPIC \
+            -s ALLOW_MEMORY_GROWTH=1 \
+            -s FORCE_FILESYSTEM=1 \
+            -s LINKABLE=1 \
+            -s USE_SDL=0 \
+            -s SIDE_MODULE=1 \
+            -s MODULARIZE=1 \
+            -s WASM_BIGINT=1 \
+            '-DARB_PREFIX="/dev/null"' \
+            '-DARB_BINARY="/dev/null"' \
+            '-DARB_LIB="/dev/null"' \
+            '-DARB_CXX_COMPILER="/dev/null"' \
```

Comment:
Same question. I guess these are ignored, but then I'd like to know why ;)

Reply:
Good one, comments comments to explain would be a good idea. The point is that these are all required to be given to the compiler or the compile will fail - yet these make no sense here as we're not installing in a prefix or a site-packages directory